src/pages/DetailMain/DetailMain.js

```diff
@@ -0,0 +1,92 @@
+import * as S from './DetailMain.Styled';
+import React from 'react';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+export default function DetailMain() {
+  const settings = {
+    dots: true,
+    infinite: false,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+
+  return (
+    <S.DetailMain>
+      <S.DetailCarousel>
+        <Slider {...settings}>
+          {NAVBARDATA_TEXT.map(img => (
+            <S.Carousel src={img.backgroundImg} key={img.id} />
+          ))}
+        </Slider>
+      </S.DetailCarousel>
+      <S.TitleBox>
+        <S.Title>{NAVBARDATA_TEXT[0].job.title}</S.Title>
+        <S.SubTitle>{NAVBARDATA_TEXT[0].job.subTitle} </S.SubTitle>
+        <S.SubBox>응답률 높음</S.SubBox>
+        <S.SubText>서울 . 한국</S.SubText>
+        <S.Text>{NAVBARDATA_TEXT[0].job.text}</S.Text>
```

Comment:
```suggestion
        <S.Text>{NAV_BAR_DATA_TEXT[0].job.text}</S.Text>
```
UPPER_SNAKE_CASE는 단어마다 _ 언더바를 추가합니다

Reply:
통신후 변경했습니다